cypress.config.js

```diff
@@ -0,0 +1,27 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+require('dotenv').config({ path: '.env.local' });
+const { defineConfig } = require('cypress');
```

Comment:
During our meeting Wed, I was kind of surprised that some files used the ESM module syntax and others (like this config file) used commonJS. Is mixing the two a common and necessary thing? In general, will any problems arise from using different import syntax in different files?

I also thought it was cool that [the config file can use `import` without a transpiler ](https://docs.cypress.io/guides/references/configuration#Configuration-File)if you add .mjs to the file--I wish Jest had a similar feature.

Reply:
The historical reason for the mixing and matching is that Node really dragged its feet when it came to supporting ESM module syntax, so you'll often see configuration files using the CommonJS syntax despite the rest of the code base using ESM syntax. Support is better now, but still sort of weird, like what you mentioned about needing to use the `.mjs` extension.

I don't know what Jest's deal is–it really seems like something they'd be able to figure out. I want to look more into [Vitest](https://vitest.dev/) as a Jest replacement. It seems to have basically the same API, but built to run much faster and avoid some of these pain points.

Reply:
Yosevu had us run Vitest when he ran the functional programming lessons.